docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Getting Started with Pre-Aggregations
+permalink: /caching/pre-aggregations/getting-started
+category: Caching
+menuOrder: 2
+---
+
+Often at the beginning of an analytical system's lifecycle - when there is a
```

Comment:
What do you think of changing "analytical system" → "analytical application" universally? I can't recall us talking about "analytical systems" anywhere in the docs and other materials we have...